ci/vm.sh

```diff
@@ -283,7 +283,7 @@ run_vm() {
 	esac
 
 	# shellcheck disable=SC2068
-	"qemu-system-$arch" \
+	exec "qemu-system-$arch" \
```

Comment:
Since `exec` from a non-main function is exotic and unexpected - can you add a  one-line comment to `run_vm()` describing that this function will never return?

Reply:
Which isn't that weird when you think about it. Bash/sh already runs the function in a subshell if there's any external calls so the `exec` just take's over the func's process.

```ShellSession
12:33  ~
$ declare -f f
f () 
{ 
    echo f:ppid:$$;
    g
}

12:33  ~
$ declare -f g
g () 
{ 
    /usr/bin/env bash -c 'echo g:ppid:$$'
}

12:33  ~
$ echo $$
64465

12:33  ~
$ f
f:ppid:64465
g:ppid:64802
```

Reply:
Actually nm, my example isn't actually relevant :D.

Reply:
I'm not sure exactly *what* causes bash to run `run_vm` in a subprocess but it sure does and the exec takes over that sub process. I'll add a little bit of docs to the func and maybe change from:

```bash
run_vm() { ... }
```
to 
```bash
run_vm() ( ... )
```
to ensure its run in a subprocess

Reply:
Added some comments to the function and changed to `run_vm() (...)` form. Better?